docs/index.rst

```diff
@@ -0,0 +1,21 @@
+.. django-fsm-log documentation master file, created by
+   sphinx-quickstart on Fri Jan 14 15:59:49 2022.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to django-fsm-log's documentation!
+==========================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+.. include:: ../README.md
+   :parser: myst_parser.sphinx_
+
+Indices and tables
+==================
+
+* :ref:`genindex`
+* :ref:`modindex`
+* :ref:`search`
```

Comment:
Unfortunately I have no experience in setting up RTD, but I think when I look at https://django-fsm-log--114.org.readthedocs.build/en/114/ that `modindex` does not and won't exist